index.js

```diff
@@ -77,6 +78,12 @@ function inferNameAndVersionFromInstalled (packageName, opts, result, cb) {
     debug('Inferring application name from productName or name in package.json')
     opts.name = result.values.productName
   }
+
+  if (result.values.version) {
+    debug('Inferring packageVersion from version in package.json')
+    opts.packageVersion = result.values.version
```

Comment:
That would make is a little confusing to me, since that would then have both a dasherized `app-version` and camelcased `appVersion` in the `opts` object.

It would seem better to fix them all as a group via #325